pages/common/7za.md

```diff
@@ -0,0 +1,19 @@
+# 7za
+
+> A file archiver with highest compression ratio
+
+- package and compress directory or file
+
+`7za a {{compressed.7z}} {{directory_or_file}}`
+
+- decompress an existing 7z file
+
+`7za x {{compressed.7z}}`
+
+- compress to zip format
+
+`7za a  -tzip {{compressed.zip}} {{directory_or_file}}`
+
+- create multipart 7zip file
+
+`7za -v{{size}}{{[b|k|m|g]}} {{compressed.7z}} {{directory_or_file}}`
```

Comment:
some clarifications committed: create multipart 7zip file; part_size: specifies part size in Bytes, Kilobytes, Megabytes or Gigabytes.
may be the comment is too long.